src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import classNames from 'classnames';
+
+interface Props {
+  isTempTodo: Todo | null;
```

Comment:
```suggestion
  todo: Todo;
```
You shouldn't render `TodoItem` at all if you don't have one. And it's not a flag, so the name is confusing